docs/index.md

```diff
@@ -56,10 +56,10 @@ Currently, Pulse provides a push-based integration with your workflow using one
 
 Share the Pulse dashboards with your teammates to bring more visibility to the metrics.
 
-To do this, copy the link on the **Organization members** page and share it with your teammates. A convenient way to do this is to share and pin the link on your team's chat.
+The most convenient way of ensuring that all the members of your company have access to the Pulse dashboards is by enabling the feature **Joining via email domain** on the **Organization members** page. When this option is enabled, anyone who signs up to Pulse using an email address from your company is automatically added to your Pulse organization.
 
-![Inviting teammates to Pulse](images/inviting-members.png)
+Alternatively, copy the invitation link from the **Organization members** page and share the link with specific people. A convenient way to do this is to share and pin the link on your team's chat. The link is valid for 3 months, but you can refresh it at any moment.
 
-By doing this, the people you invite will have access to all dashboards immediately after signing up without needing to perform any integration.
+![Adding organization members](images/joining-email-domain.png)
 
-The link is valid for 3 months, but you can refresh it at any moment.
+The people who sign up using your company's email addres or that you explicitly invite will have access to all dashboards immediately after signing up without needing to perform any integration.
```

Comment:
> The people who sign up using your company's email **address** or that you explicitly invite

I found this a little long, but I understand the reasoning. We could try something like "New members", "Newcomers", or "People who signup via email domain or invitation link"

Just a couple of thoughts. <3 Everything is looking really clear ☀️

Reply:
I like this suggestion, sometimes it happens that I'm being very precise but a more conversational style works best. :ok_hand: